relay-config/src/config.rs

```diff
@@ -904,8 +904,10 @@ struct Cache {
     envelope_buffer_size: u32,
     /// The cache timeout for non-existing entries.
     miss_expiry: u32,
-    /// The buffer timeout for batched queries before sending them upstream in ms.
+    /// The buffer timeout for batched project config queries before sending them upstream in ms.
```

Comment:
Since `batch_interval` is only used in the Project Upstream service, we can be more explicit in the docs.